dspace-api/src/main/java/org/dspace/authorize/ResourcePolicyServiceImpl.java

```diff
@@ -235,6 +241,10 @@ public void removeAllPolicies(Context c, DSpaceObject o) throws SQLException, Au
         c.turnOffAuthorisationSystem();
         contentServiceFactory.getDSpaceObjectService(o).updateLastModified(c, o);
         c.restoreAuthSystemState();
+        if (o instanceof Item) {
```

Comment:
Duplicity in the code

Reply:
Do you want to remove only duplicates of this type in this file, or in other files as well?